src/modules/oh-indicators-list/store/slice.ts

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { fetchAllItems } from '../network-layer';
+import { IOpenhabItem } from '../types/openHab';
+
+export interface IState {
+  items: IOpenhabItem[];
+  error: string | null;
+}
+
+export const getAllItems = createAsyncThunk<IOpenhabItem[], void>(
+  'items/fetchAllItems',
+  fetchAllItems,
+);
+
+export const indicatorsSlice = createSlice<IState, any>({
+  name: 'persistentStateSlice',
+  initialState: {
+    items: [],
+    error: null,
+  },
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(getAllItems.fulfilled, (state, action: PayloadAction<IOpenhabItem[]>) => {
+      state.items = state.items.concat(action.payload);
```

Comment:
@chooidos think about which data structure is easier to access by an element id? Is array (current) good for that?